CHANGELOG.md

```diff
@@ -6,6 +6,14 @@
 
 ## Breaking Changes
 
+# V7.7.0
```

Comment:
Are we missing the new Changes since?

Reply:
Indeed we are. Let quickly check the release pipeline.

Reply:
Fixed it in the current CHANGELOG and added the necessary logic in the pipeline to automatically generate the section.